seminar8/base.py

```diff
@@ -0,0 +1,127 @@
+import os
+import csv
+import pandas
```

Comment:
У вас показываются не все столбцы. Сделайте так, чтобы показывались все:
```suggestion
import pandas
pandas.set_option('display.max_columns', None)
pandas.set_option('display.width', 200)
```